desktop/app/main.js

```diff
@@ -11,6 +11,8 @@ const installer = require('./installer')
 const app = require('electron').app
 const path = require('path')
 const showDevTools = require('../../react-native/react/local-debug.desktop').showDevTools
+const shell = require('electron').shell
```

Comment:
small thing, this isn't really 'help' per se but like first-run-milestone-2 help. dunno if we wanna rename it or if it matters